### ISSUE TITLE
Regular expression route not working

### ISSUE THREAD

**Issue by silviubogan** [IonicaBizau/statique#42]
I have this code:

``` js
var Statique = require("statique"),
    Http = require("http");

var server = new Statique({
    root: __dirname + "/public",
    cache: 36000
}).setRoutes({
    "\/anynumber\-[0-9]": {
        re: true,
        get: function (req, res) {
            res.end("Hi");
        }
    }
}).setErrors({
    404: "/errors/404.html"
});

Http.createServer(server.serve.bind(server)).listen(8000, function (err) {
    console.log("Listening on port 8000.");
});
```

Opening [http://localhost:8000/anynumber-5](http://localhost:8000/anynumber-5) in the browser returns 404.

I am using `statique@3.2.1`.

**Comment by IonicaBizau** [comment]
Looks like it doesn't work when having the methods as fields. If you replace `get` with `handler` it works, but it should work with `get`, `post` etc as well.

Contributions are welcome! :sparkle:

**Comment by silviubogan** [comment]
Solved in 263c706 from #52..
